DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/CentreConfigurationController.cs

```diff
@@ -80,13 +90,36 @@ public IActionResult EditCentreWebsiteDetails(EditCentreWebsiteDetailsViewModel
                 return View(model);
             }
 
+            model.CentrePostcode = model.CentrePostcode!.Trim();
+            var mapsResponse = mapsApiHelper.GetPostcodeCoordinates(model.CentrePostcode).Result;
+
+            if (mapsResponse.Status == "ZERO_RESULTS")

```

Comment:
Could we pull these status codes into the maps helper or models, so we don't need to reference the strings everywhere?
For example, could we do mapsResponse.Status.isNoResults() or something similar here? :)